### PR TITLE
Beachfront: Add schain support

### PR DESCRIPTION
**Pull request by muncha**
This adds schain data to the back end banner call, and fixes test cases to match.

### CODE REVIEW

File: adapters/beachfront/beachfront.go [hunk 1]
```diff
@@ -515,9 +533,7 @@ func (a *BeachfrontAdapter) MakeBids(internalRequest *openrtb2.BidRequest, exter
 
 	// For video, which uses RTB for the external request, this will unmarshal as expected. For banner, it will
 	// only get the User struct and everything else will be nil
-	if err := json.Unmarshal(externalRequest.Body, &xtrnal); err != nil {
-		errs = append(errs, err)
-	}
+	json.Unmarshal(externalRequest.Body, &xtrnal)
```
Comment (guscarreon) [inline adapters/beachfront/beachfront.go:536]: Why are we ignoring the unmarshal error? As unlikely as it is, we believe it is better to check for it.

File: adapters/beachfront/beachfront.go [hunk 2]
```diff
@@ -569,7 +585,7 @@ func postprocess(response *adapters.ResponseData, xtrnal openrtb2.BidRequest, ur
 
 		if err := json.Unmarshal(response.Body, &beachfrontResp); err != nil {
 			return nil, []error{&errortypes.BadServerResponse{
-				Message: fmt.Sprint("server response failed to unmarshal as valid rtb. Run with request.debug = 1 for more info"),
+				Message: "server response failed to unmarshal as valid rtb. Run with request.debug = 1 for more info",
```
Comment (SyntaxNode) [inline adapters/beachfront/beachfront.go:588]: Nice catch.

File: adapters/beachfront/beachfront.go (hunk 1 above)
Comment (muncha) [reply adapters/beachfront/beachfront.go:536]: I will stop ignoring it.

**Comment by muncha** [comment]
I'm setting this as a draft for now. I've run into a few things I'd like to improve.

**Review by bsardo — APPROVED** [review]
LGTM

**Review by guscarreon — APPROVED** [review]
LGTM

**Review by guscarreon — APPROVED** [review]
LGTM

**Pull request merged**